packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -53,6 +80,139 @@ export class Wallet {
       { amount: 'bignumber' }
     )
   }
+
+  /**
+   * Create a new wallet
+   *
+   * @param walletName
+   * @param disablePrivateKeys
+   * @param options
+   * @param options.blank
+   * @param options.passphrase
+   * @param options.avoidReuse
+   * @return Promise<IWallet>
+   */
+  async createWallet (
+    walletName: string,
+    disablePrivateKeys = false,
+    options: CreateWalletOptions = {}
+  ): Promise<IWallet> {
+    const { blank = false, passphrase = '', avoidReuse = false } = options
+
+    return await this.client.call(
+      'createwallet',
+      [walletName, disablePrivateKeys, blank, passphrase, avoidReuse],
+      'number'
+    )
+  }
+
+  /**
+   * Return object containing various wallet state info
+   *
+   * @return Promise<WalletInfo>
+   */
+  async getWalletInfo (): Promise<WalletInfo> {
+    return await this.client.call('getwalletinfo', [], {
+      balance: 'bignumber',
+      unconfirmed_balance: 'bignumber',
+      immature_balance: 'bignumber',
+      paytxfee: 'bignumber'
+    })
+  }
+
+  /**
+   * Change the state of the given wallet flag for a wallet
+   *
+   * @param flag to change. eg: avoid_reuse
+   * @param value optional, default = true
+   * @return Promise<WalletFlagResult>
+   */
+  async setWalletFlag (flag: WalletFlag, value: boolean = true): Promise<WalletFlagResult> {
+    return await this.client.call('setwalletflag', [flag, value], 'number')
+  }
+
+  /**
+   * Returns a new Defi address for receiving payments.
```

Comment:
I think a mental note here, although it's called **Defi** on the ain side, we might want to enforce calling it **DeFi** from now on.